### PR TITLE
13 fix unclear comments in dnatracing

### PR DESCRIPTION
**Pull request by JoeBeton**
I edited the vague/totally useless comments in getFittedTraces() - would be good to know if the new comments are clear enough for others to understand. Can improve/clarify comments further if necessary.

### CODE REVIEW

**Comment by bobturneruk** [comment]
If I write resolves #24, then issue 24 will be automatically closed when this is merged!

File: dnatracing.py [hunk 1]
```diff
@@ -186,33 +188,56 @@ def reportBasicStats(self):
...
+                # e.g. indexing self.gauss_image[130, 130] for 128x128 image
                 if trace_coordinate[0] < 0:
-                    trace_coordinate[0] = height_search_distance
-                elif trace_coordinate[0] >= self.number_of_rows - height_search_distance:
-                    trace_coordinate[0] = trace_coordinate[0] = self.number_of_rows - height_search_distance
+                    # prevents negative number indexing
+                    # i.e. stops (trace_coordinate - index_width) < 0
+                    trace_coordinate[0] = index_width
+                elif trace_coordinate[0] >= self.number_of_rows -
+                                                        index_width:
```
Comment (Jean-Du) [inline dnatracing.py:227]: Move this to the line above?

File: dnatracing.py [hunk 2]
```diff
@@ -186,33 +188,56 @@ def reportBasicStats(self):
...
+                    # prevents indexing above image range causing IndexError
+                    trace_coordinate[0] = self.number_of_rows -
+                                                        index_width
```
Comment (Jean-Du) [inline dnatracing.py:230]: Again, should this be moved to the line above?

File: dnatracing.py [hunk 3]
```diff
@@ -228,40 +253,67 @@ def getFittedTraces(self):
...
+                                        trace_coordinate[1] + index_width
+                                        )
                     x_coords = np.full(len(y_coords), trace_coordinate[0])
 
-                #Use the perp array to index the guassian filtered image
+                # Use the perp array to index the guassian filtered image
                 perp_array = np.column_stack((x_coords, y_coords))
-                height_values = self.gauss_image[perp_array[:,1],perp_array[:,0]]
+                height_values =
+                            self.gauss_image[perp_array[:,1], perp_array[:,0]]
```
Comment (Jean-Du) [inline dnatracing.py:311]: Why is this put into two lines?

**Review by Jean-Du — CHANGES_REQUESTED** [review]
There are a few occasions where one line seems to be separated into two; was this intended?
Also regarding line 315, "can delete", is this part still used by the code? Might want to test by commenting it out and see if it still works before saying it can be deleted

**Comment by JoeBeton** [comment]
> 
> 
> There are a few occasions where one line seems to be separated into two; was this intended?
> Also regarding line 315, "can delete", is this part still used by the code? Might want to test by commenting it out and see if it still works before saying it can be deleted

Python style guide (PEP 8) specified a maximum line length of 79 characters. Not the biggest issue in the world but does make the code more readable if the reader doesn't have to horizontally scroll to see a full line of code. 

See specific part of Python style guide here: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

**Review by Jean-Du — APPROVED** [review]

**Pull request merged**